<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DentalCare</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="header" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #5FA8D3; color: white;</string>
    </property>
    <widget class="QLabel" name="clinicName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DentalCare</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogout">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FF0000; color: white;</string>
     </property>
     <property name="text">
      <string>Cerrar Sesión</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sidebar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>200</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #E0F7FF;</string>
    </property>
    <widget class="QPushButton" name="btnUsers">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestión de Usuarios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDentists">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestión de Dentistas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAppointments">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Citas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRecords">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Expedientes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainContent" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>600</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;</string>
    </property>
    <widget class="QLabel" name="welcomeImage">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="pixmap">
      <pixmap>logo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="welcomeMessage">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>400</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Bienvenido a DentalCare</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
